pkg/controller/routing.go

```diff
@@ -30,6 +30,10 @@ import (
 	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
 )
 
+const (
```

Comment:
Can we move this constant to `controller/constants.go`

Reply:
Moved it to controller/constants.go